api/turing/webhook/webhook.go

```diff
@@ -0,0 +1,73 @@
+package webhook
+
+import (
+	"context"
+
+	"github.com/caraml-dev/mlp/api/pkg/webhooks"
+
+	"github.com/caraml-dev/turing/api/turing/models"
+)
+
+type Client interface {
+	TriggerRouterEvent(ctx context.Context, eventType webhooks.EventType, router *models.Router) error
+	TriggerEnsemblerEvent(ctx context.Context, eventType webhooks.EventType, ensembler models.EnsemblerLike) error
```

Comment:
what happen if this method is called with eventType `OnRouterDeployed` ?

Reply:
Fixing in https://github.com/caraml-dev/turing/pull/395/commits/7760b00e620e5386fa545f67363f000043a48e2f